docs/components/modeler/web-modeler/diagram-errors.md

```diff
@@ -0,0 +1,33 @@
+---
+id: diagram-errors
+title: Diagram errors
+description: This feature will assist you in debug and fix errors in your processes.
```

Comment:
I think we should establish an absolute minimum set of terms we use for this feature.

As a User reading this, I might get very confused when reading: "lint errors", "engine errors", "debug panel" etc.

I would rather narrow the terms down to two types of errors:
* Lint errors
* Deploy errors

And since the panel will always display errors, then use the
* Diagram error panel
term for the component. Also, why call it "debug panel" if the title on the component is "Diagram errors"?
We should ensure that the terms in the UI are aligned with the documentation.